state.tf

```
@@ -2,15 +2,34 @@ provider "azurerm" {
   features {}
 }
 
+provider "azurerm" {
+  alias           = "aks-infra"
+  subscription_id = "${var.aks_subscription_id}"
```

applies throughout, string interpolation not needed if it's just a variable, it will be causing lots of warnings like this
```suggestion
  subscription_id = var.aks_subscription_id
```

---

yep 84 of them